relay/nakama/signer/kms.go

```diff
@@ -134,7 +127,7 @@ func calculateDigest(tx *sign.Transaction) common.Hash {
 	return crypto.Keccak256Hash(
```

Comment:
Same as my comments in `sign`, is `created` also included in the payload to sign?

Reply:
The comment for the function suggests that we should be using sign.Transaction.PopulateHash() instead of duplicating it here, and that makes sense. So I fixed that and removed calculateDigest altogether.